### PR TITLE
ENG-1402 implement incremental name and code feature

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/pages/add/PagesAddFormContainer.js
Comment (gidesan) [inline src/ui/pages/add/PagesAddFormContainer.js:22]: can you please make it more functional?

**Review by gidesan — CHANGES_REQUESTED** [review]

File: src/ui/pages/add/PagesAddFormContainer.js
Comment (gidesan) [reply src/ui/pages/add/PagesAddFormContainer.js:22]: I'll made some commits to show up what I mean (and also something to improve test readability)

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/906#pullrequestreview-525371911" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
